lib/membrane_mp4/demuxer/isom/sample_data.ex

```diff
@@ -0,0 +1,205 @@
+defmodule Membrane.MP4.Demuxer.ISOM.SampleData do
+  @moduledoc false
```

Comment:
This module seems to be doing some wild things, yet it doesn't have even a single doc/comment.